exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,33 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number) {
+  if (number < 0){
+    return true;
+  }
+  return false;
+}
+function isBetween5and10(number) {
+  if (5<number < 10) {
```

Comment:
I don't think this expression will return false if number is outside the range. Try using two expressions and join them with `&&`